### ISSUE TITLE
Use twbs:bootstrap instead of mizzao version

### ISSUE THREAD

**Issue by ghost** [andruschka/meteor-bootstrap-paper#1]
The repo at https://github.com/mizzao/meteor-bootstrap-3 recommends us to use twbs version which is the official version published by bootstrap developers.

Please add twbs:bootstrap as dependency instead of mizzao's version. 
Even most of the packages are making twbs as dependency. This would help in de-duplication of bootstrap packages as well.

Thanks.

**Comment by andruschka** [comment]
Yes of course - had other thigs around the last few weeks - all packages will get a refresh soon...

**Comment by dandv** [comment]
Hi @bluebay / andruschka,

How is this package different from other two on Atmosphere that do the same thing?
- @iamthemuffinman's [bootstrap-paper](https://github.com/iamthemuffinman/meteor-bootstrap-paper/issues/1)
- @fxv's [bootstrap-paper](https://github.com/fxv/bootstrap-paper/issues/1)

Both packages have problems, so I recommend that you gentlemen join efforts and create _one_ properly packaged wrapper, modeled after https://github.com/MeteorPackaging/webix-skin-flat/.

Obsolete package should be hidden from Atmosphere searches with

```
meteor admin set-unmigrated <user>:<package-name>
```

**Comment by dandv** [comment]
Nevermind, see #1.

**Comment by dandv** [comment]
@bluebay: see the [official Bootswatch packages](http://atmospherejs.com/bootswatch).
